containers/signaturetoolslib/HRDetect_wrapper.R

```diff
@@ -185,6 +186,21 @@ input_matrix[rownames(Indel.del.mh.prop),"del.mh.prop"] <- Indel.del.mh.prop[,"d
 # Run the pipeline
 message("")
 message("========= Running HRDetect Pipeline =========")
+hrd.arglist = list()
+hrd.arglist[["data_matrix"]] = input_matrix
+hrd.arglist[["genome.v"]] = genome_version
+hrd.arglist[["SV_bedpe_files"]] = SV_bedpe_files
+hrd.arglist[["Indels_tab_files"]] = Indels_tab_files
+hrd.arglist[["CNV_tab_files"]] = CNV_tab_files
+hrd.arglist[["SNV_catalogues"]] = SNV_catalogues
+hrd.arglist[["nparallel"]] = n_parallel
+if (strsplit(as.character(packageVersion("signature.tools.lib")),"\\.")[[1]][1] %in% c("0","1")){
+	hrd.arglist[["signature_type"]] = "COSMIC"
+} else {
+	hrd.arglist[["SNV_signature_version"]] = "COSMICv2"
+}
+res <- do.call(HRDetect_pipeline, hrd.arglist)
+if (1==0) {
```

Comment:
What is this?

Reply:
Good catch! I forgot to clean up. Line 202 will run HRDetect_pipeline, while 203-212 is the the old code that I can  just remove.